timer.go

```diff
@@ -0,0 +1,137 @@
+// Mgmt
+// Copyright (C) 2013-2016+ James Shubin and the project contributors
+// Written by James Shubin <james@shubin.ca> and the project contributors
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+package main
+
+import (
+	"encoding/gob"
+	"log"
+	"time"
+)
+
+func init() {
+	gob.Register(&TimerRes{})
+}
+
+//TimerRes is a timer resource
+type TimerRes struct {
+	BaseRes  `yaml:",inline"`
+	Interval int          `yaml:"interval"` //Interval : Interval between runs
+	Ticker   *time.Ticker //Ticker
+}
+
+type TimerUUID struct {
+	BaseUUID
+	name string
+}
+
+func NewTimerRes(name string, interval int, command string) *TimerRes {
+	obj := &TimerRes{
+		BaseRes: BaseRes{
+			Name: name,
+		},
+		Interval: interval,
+		Ticker:   time.NewTicker(time.Duration(interval) * time.Second),
+	}
+	obj.Init()
+	return obj
+}
+
+func (obj *TimerRes) Init() {
+	obj.BaseRes.kind = "Timer"
+	obj.BaseRes.Init() //call base init, b/c we're overrriding
```

Comment:
You clearly copied and pasted from:
```
	obj.BaseRes.Init() // call base init, b/c we're overriding
```
which is fine, but somehow you managed to get rid of the space after the //   .... why and how???